node/clients/product.ts

```diff
@@ -31,7 +31,7 @@ export class Products extends JanusClient {
   }
 
   public async getProductsName(workspace: string, ids: number[]) {
-    const names: string[] = []
+    const names: Array<{ id: string; name: string }> = []
```

Comment:
I suggest create a interface to use in this and other cases that use the same structure, you know?

```suggestion
    interface Name {
         id: string
         name: string
    }
    ...
    const names: Array<Name> = []
```

or

```suggestion
    interface Name {
         id: string
         name: string
    }
    ...
    const names: Name[] = []
```